src/nginx/module.h

```diff
@@ -248,6 +248,11 @@ struct ngx_esp_request_ctx_s {
 
   // Nginx variable $backend_url. set from service config.
   ngx_str_t backend_url;
+
+  // If gRPC response have a "grpc-status-details-bin" header, which
```

Comment:
Update the comment as
// Stores proto message of gRPC response header "grpc-status-details-bin"

Reply:
Done.